config/locales/de/models.yml

```diff
@@ -6,6 +6,7 @@ de:
         api_key: API-Schlüssel
         check_uuid_url: URL zum Überprüfen der UUID
         name: Name
+        proforma_version: Proforma Version
```

Comment:
Could you please double-check the spelling of "ProFormA"? I would like to have a single, common spelling used throughout the app: "ProFormA".

For compound words, please use "ProFormA-XML" (actually, only in German, but since the group uses the same term with `-` in English too, let's reuse it in both languages). But, if possible, I would drop the "-XML" suffix, since this can create really misleading wordings: "Es ist ein Fehler beim Importieren der ProformaXML ZIP-Datei aufgetreten."